WORKSPACE

```diff
@@ -103,6 +103,27 @@ stack_snapshot(
     },
 )
 
+new_local_repository(
+    name = "system-libs",
+    # pkg-config --variable=libdir x11
```

Comment:
Seems like a leftover?

Reply:
Oh I am sorry, don't think this is ready for review yet. It fixes the problem, but the test case requires a local library which isn't portable.

Reply:
I converted the PR into a draft, feel free to undo that once it's ready :)